test/functional/interface_zmq.py

```diff
@@ -372,7 +372,6 @@ def test_sequence(self):
         # since it greatly depends on inner-workings of blocks/mempool
         # during "deep" re-orgs. Probably should "re-construct"
         # blockchain/mempool state from notifications instead.
-        block_count = self.nodes[0].getblockcount()
```

Comment:
Dead Python code should be getting caught by Vulture, see https://github.com/bitcoin/bitcoin/blob/master/test/lint/lint-python-dead-code.py. Maybe we need to drop the minimum confidence.

Reply:
We can decrease confidence. But [this](https://github.com/bitcoin/bitcoin/blob/master/test/lint/lint-python-dead-code.py#L29) line worries me a little.

> Any value below 100 introduces the risk of false positives, which would create an unacceptable maintenance burden.

Can we afford false positives for the chance of catching the few missed cases?

Reply:
> Can we afford false positives for the chance of catching the few missed cases?

It should either be tweaked to the point that it catches obviously dead code, or just removed. No point running a dead code linter that doesn't catch dead code.

Reply:
> No point running a dead code linter that doesn't catch dead code.

I agree with it.